src/main/java/uk/gov/ons/census/fwmt/jobservice/rabbit/RmReceiver.java

```diff
@@ -41,10 +41,12 @@ public RmReceiver(JobService jobService, GatewayEventManager gatewayEventManager
   }
 
   @RabbitHandler
-  public void receiveCreateMessage(FwmtActionInstruction rmRequest, Message message) throws GatewayException {
+  public void receiveCreateMessage(FwmtActionInstruction rmRequest, @Header("timestamp") String timestamp, Message message) throws GatewayException {
     //TODO trigger correct event CREATE or UPDATE
-    Instant receivedMessageTime =message.getMessageProperties().getTimestamp().toInstant();
+    long epochTimeStamp = Long.parseLong(timestamp);
+    Instant receivedMessageTime = Instant.ofEpochMilli(epochTimeStamp);
     System.out.println(receivedMessageTime);
+    System.out.println(message.getMessageProperties());
```

Comment:
Are you sure this will printout the properties, have you tried locally
might be worth printing out the ke/values in the message.getHeaders map